.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
+.venv
+result.png
+checklist.md
+.gitignore
```

Comment:
Ensure .gitignore is not needed in the Docker image. Typically, it's only needed for version control and not for the application runtime.